mix.exs

```diff
@@ -97,7 +105,7 @@ defmodule Sentry.Mixfile do
       {:hackney, "~> 1.8", optional: true},
       {:jason, "~> 1.1", optional: true},
       {:phoenix, "~> 1.6", optional: true},
-      {:phoenix_live_view, "~> 0.20", optional: true},
+      {:phoenix_live_view, live_view_version(), optional: true},
```

Comment:
I think instead of basing on the Elixir version, we should do `~> 0.20 or ~> 1.0` here. Thoughts?

Reply:
@whatyouhide despite the fact that LV 1.0.0 does not work with older Elixir, it did happily install 1.0.0 on Elixir 1.13 - so I thought it has to be explicitly restricted by the version requirement.

Reply:
Ok, but we still can't bump the version to `~> 1.0` for Sentry (when Elixir >= 1.14) because people might be depending on older LV versions and Sentry doens't need the latest.

Reply:
Why not just change it to `:phoenix_live_view, ">= 0.20", optional: true`

Reply:
`>= 20.0` will have the same issue as `~> 0.20 or ~> 1.0`

Reply:
Maybe I'm being dumb, but wouldn't `>= 0.20` open up for consumers to use LiveView 1.0 where the current `~> 0.20` is equivalent to `>= 0.20 and < 0.21`.

To be clear I'm currently blocked on upgrading to LiveView 1.0 because of this:

```
% mix deps.get
Resolving Hex dependencies...
Resolution completed in 0.167s
Because sentry >= 10.5.0 depends on phoenix_live_view ~> 0.20 and your app depends on phoenix_live_view ~> 1.0, sentry >= 10.5.0 is forbidden.
So, because your app depends on sentry ~> 10.7, version solving failed.
** (Mix) Hex dependency resolution failed
```